bedrock/mozorg/templates/mozorg/robots.txt

```diff
@@ -6,5 +6,9 @@ disallow: /*/download/
 disallow: /*/firstrun/
 disallow: /*/newsletter/existing/
 disallow: /*/whatsnew/
+disallow: /*/firefox/os/1.1/
```

Comment:
@pmclanahan - added these to `robots.txt` as suggested. Just checking these look ok?